source/repo_specific_helpers.rb

```diff
@@ -0,0 +1,7 @@
+def saveGrammar(grammar)
```

Comment:
shoulden't this be part of textmate_tools?

Reply:
Ideally textmate_tools.rb will become library/ruby gem and mapping it to a specific folder structure would kind of break that.